### PR TITLE
Try to clarify some doubts

### PR DESCRIPTION
**Pull request by alexpdp7**
I had a conversation with someone who was confused by our terms- this tries to address this.

I'll send him the PR so he can weigh in if he wants, and note if this solves his issue; so let's wait prudentially before merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,15 @@ We will only list companies which allow working fully remotely (e.g. you can
...
 don't want to). Some occasional travelling for onboarding, team building or
 similar purposes is fine. Also the companies must also hire people using
-Spanish full-time contracts (i.e. not self-employment).
+Spanish full-time contracts (i.e. not self-employment). For example, the
+following scenarios are OK:
+
+* A completely Spanish company that hires in Spain with regular, non
+  self-employed contracts people living in Spain
+* A foreign company, with a Spanish legal entity to be able to hire people
+  living in Spain, using regular, non-self employed contracts
```
Comment (uesteibar) [inline README.md:38]:
```suggestion
  living in Spain, using regular, non self-employed contracts
```

To use the same format as the point above

**Review by uesteibar — APPROVED** [review]
LGTM. What was the confusion if I may ask? Maybe it's something we can address in some other way too

**Comment by alexpdp7** [comment]
I think he was worried about we not specifying if this is for local or foreign companies. I think they were confused because their company hires people outside Spain too, as contractors.

I don't think it's too bad, but adding a few examples shouldn't hurt I guess...

**Comment by alexpdp7** [comment]
(edit: I'll leave this open and merge tomorrow COB or something like that, I guess- doesn't sound critical)

**Pull request merged**